sample/android-app/app/build.gradle.kts

```diff
@@ -51,6 +51,8 @@ android {
 }
 
 marathon {
+    analyticsTracking = false
```

Comment:
Please remove these from the sample app. Instead, maybe the docs need an example of disabling bugsnag for both CLI and for Gradle, currently there are none.

Reply:
Removed and updated `docs/docs/intro/configure.md`